go/host/enclave/guardian.go

```diff
@@ -470,14 +470,24 @@ func (g *Guardian) processL1BlockTransactions(block *common.L1Block) {
 		if err != nil {
 			g.logger.Error("Could not decode rollup.", log.ErrKey, err)
 		}
-		err = g.db.AddRollupHeader(r, block)
+
+		metaData, err := g.enclaveClient.GetRollupData(r.Header.Hash())
+		if err != nil {
+			g.logger.Error("Could not fetch rollup metadata from enclave.", log.ErrKey, err)
+		}
+		err = hostdb.AddRollup(g.db, r, metaData, block)
 		if err != nil {
 			if errors.Is(err, errutil.ErrAlreadyExists) {
 				g.logger.Info("Rollup already stored", log.RollupHashKey, r.Hash())
 			} else {
 				g.logger.Error("Could not store rollup.", log.ErrKey, err)
 			}
 		}
+		// TODO (@will) need to store this data? For now we only store the blocks with rollups
```

Comment:
Nice, yeah wouldn't have thought we'd need to hold data about blocks that had rollups in. We should have put all relevant data for future lookups in the rollups table.

Reply:
This is just to preserve backwards compatibility with tenscan which provides a block list - will remove this once the tenscan UI is updated to show rollups instead of block data